include/smac_planner/smac_planner_hybrid.hpp

```diff
@@ -78,6 +79,26 @@ class SmacPlannerHybrid : public mbf_costmap_core::CostmapPlanner
     double & cost,
     std::string & message)  override;
 
+
+  /**
+   * @brief Creating a direct path from start to goal pose.
+   * @param start Start pose
+   * @param goal Goal pose
+   * @param tolerance If the goal is obstructed, how many meters the planner can relax the constraint
+   *        in x and y before failing
+   * @param plan The plan... filled by the planner
```

Comment:
S: no need to add more comments in this case, 
maybe just 
`@param plan vector of PoseStamped`